public/components/overview/components/select_dashboard_flyout.tsx

```diff
@@ -86,25 +91,59 @@ export function SelectDashboardFlyout({ closeFlyout, dashboardsSavedObjects, rel
         </EuiText>
       </EuiFlyoutHeader>
       <EuiFlyoutBody className="selectable-flyout-body">
-        <EuiSelectable
-          searchable
-          searchProps={{
-            placeholder: 'Search for a dashboard...',
-            compressed: true,
-          }}
-          singleSelection="always"
-          options={options}
-          onChange={onSelectionChange}
-          listProps={{ bordered: false }}
-          height="full"
-        >
-          {(list, search) => (
-            <>
-              {search}
-              {list}
-            </>
-          )}
-        </EuiSelectable>
+        {Object.keys(dashboardsSavedObjects).length === 0 ? (
```

Comment:
just for my knowledge, what is this check for? Could you elaborate more of this scenario?

Reply:
If a user does not have a dashboard to set, we want to direct them in a path to create one.
<img width="389" alt="Screenshot 2024-10-18 at 12 25 45 PM" src="https://github.com/user-attachments/assets/dc70d348-b95c-4eec-af86-b18aaa751864">
<img width="391" alt="Screenshot 2024-10-18 at 12 26 26 PM" src="https://github.com/user-attachments/assets/5bdbd1b8-9235-40d4-8ea7-fb4b04ca384d">